examples/numalogic-simple-pipeline/ml_steps/udf/inference.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+
+import numpy as np
+from numalogic.models.autoencoder import AutoencoderPipeline
+from numalogic.models.autoencoder.variants import VanillaAE
+from pynumaflow.function import Messages, Message, Datum
+
+from ml_steps.utility import Payload, load_model
+
+LOGGER = logging.getLogger(__name__)
+DEFAULT_WIN_SIZE = 12
+WIN_SIZE = int(os.getenv("WIN_SIZE", DEFAULT_WIN_SIZE))
+
+
+def inference(key: str, datum: Datum) -> Messages:
+    # Load data and convert bytes to MessagePacket
+    payload = Payload.from_json(datum.value.decode("utf-8"))
+
+    messages = Messages()
+
+    artifact = load_model()
+    try:
+        # load model from registry
+        pl = AutoencoderPipeline(model=VanillaAE(WIN_SIZE), seq_len=WIN_SIZE)
```

Comment:
Should we consider to save all model parameters in the model store?

Those params include:
- what ML to use (AutoencoderPipeline in this case)
- which variant to use (VanillaAE in this case)
- other hyperparams, including WIN_SIZE here.